apis/apps/v1alpha1/apicast_types.go

```diff
@@ -466,3 +466,91 @@ func (a *APIcast) Validate() field.ErrorList {
 func init() {
 	SchemeBuilder.Register(&APIcast{}, &APIcastList{})
 }
+
+func (a *APIcast) GetAdminPortalCredentialsSecretRef() *v1.LocalObjectReference {
+	return a.Spec.AdminPortalCredentialsRef
+}
+
+func (a *APIcast) GetEmbeddedConfigurationSecretRef() *v1.LocalObjectReference {
+	return a.Spec.EmbeddedConfigurationSecretRef
+}
+
+func (a *APIcast) GetOpenTelemetrySecretRef() *v1.LocalObjectReference {
+	if a.Spec.OpenTelemetry == nil {
+		return nil
+	}
+	return a.Spec.OpenTelemetry.TracingConfigSecretRef
+}
+
+func (a *APIcast) GetOpenTracingSecretRef() *v1.LocalObjectReference {
```

Comment:
this will be dropped in 2.16 so feel free to skip any openTracing stuff

Reply:
If there's no harm in keeping it then I might just leave it in. But let me know if it needs to be removed.